src/registrar/templates/portfolio_requests.html

```diff
@@ -4,7 +4,12 @@
 
 {% block title %} Domain requests | {% endblock %}
 
+{% block wrapper_class %}
+  {{ block.super }} dashboard--grey-1
+{% endblock %}
+
 {% block portfolio_content %}
+<div id="main-content">
```

Comment:
Is main-content used mostly for organization of the templates? I don't see any styling references within the codebase.

Reply:
It's used as a target for the skip nav functionality, which is an accessibility requirement

Reply:
Thank you for clarifying!